client/lib/abtest/active-tests.js

```diff
@@ -81,4 +81,12 @@ module.exports = {
 		defaultVariation: 'drake',
 		allowExistingUsers: true,
 	},
+    swapButtonsMySiteSidebar: {
+	    datestamp: '20160414',
+	    variations: {
+            original: 50,
+            swap: 50
+        },
+        defaultVariation: 'original'
+    }
```

Comment:
Very minor, but if you end this with a comma then it will be easier to add tests in the future.